velox/docs/functions/spark/array.rst

```diff
@@ -126,6 +126,14 @@ Array Functions
     if :doc:`spark.legacy_size_of_null <../../configs>` is set to false.
     Otherwise, returns -1 for null input.
 
+.. spark:function:: slice(array(E), start, length) -> array(E)
+
+    Returns a subarray starting from index ``start``(array indices start at 1, or starting from the end
```

Comment:
This statement is long. Please remove parentheses and rephrase the sentences.

Suggestion:
"array indices start at 1" -> the ``start`` index is 1-based.